### PR TITLE
Fix(lsbackup): Fix profiler in lsBackup (#7729)

### PR DESCRIPTION
**Pull request by dshekhar95**
cherry pick from #7729

## Problem
wrong config set for profiler (set to Restore instead of Backup)

## Solution
set profiler to Backup.conf

### CODE REVIEW

**Review by meghalims — DISMISSED** [review]
Hi Dilip, as we have discussed, please add unit test for backup/run.go.

File: ee/backup/run.go [hunk 1]
```diff
@@ -42,6 +43,7 @@ import (
 )
 
 // Restore is the sub-command used to restore a backup.
+
```
Comment (akon-dey) [inline ee/backup/run.go:46]: This is an unnecessary change.

File: ee/backup/run.go [hunk 2]
```diff
@@ -162,7 +164,7 @@ func initBackupLs() {
 		Short: "List info on backups in a given location",
 		Args:  cobra.NoArgs,
 		Run: func(cmd *cobra.Command, args []string) {
-			defer x.StartProfile(Restore.Conf).Stop()
+			defer x.StartProfile(LsBackup.Conf).Stop()
```
Comment (akon-dey) [inline ee/backup/run.go:167]: LGTM

File: ee/backup/run.go (hunk 1 above)
Comment (dshekhar95) [reply ee/backup/run.go:46]: @akon-dey the var Restore x.SubCommand on line 47 is used in other part of the code. So I had to re-introduce to fix the errors

Comment (dshekhar95) [reply ee/backup/run.go:46]: looks like github isnt showing the line 47 addition unless you click view changes, not sure if you missed that

**Comment by dshekhar95** [comment]
> 

This will need an backup integration test

**Comment by anurags92** [comment]
Please add some details about the change in the PR description.

**Review by matthewmcneely — APPROVED** [review]

**Review by dshekhar95 — COMMENTED** [review]
fixed

**Review by meghalims — APPROVED** [review]
Looks good. Please merge

**Comment by damonfeldman** [comment]
This fix allows profiling of the Backup operation to work properly if certain profilng flags are specified in the configuration. Previously, the system was profiling restore instead of backup.
